src/Kros.AspNetCore/Extensions/ConfigurationBuilderExtenstions.cs

```diff
@@ -14,5 +20,41 @@ public static class ConfigurationBuilderExtenstions
         /// <returns>The same instance of the Microsoft.Extensions.Hosting.IHostBuilder for chaining.</returns>
         public static IConfigurationBuilder AddLocalConfiguration(this IConfigurationBuilder configurationBuilder)
             => configurationBuilder.AddJsonFile("appsettings.local.json", optional: true);
+
+        /// <summary>
+        /// Adds the azure application configuration.
+        /// </summary>
+        /// <param name="config">The configuration.</param>
+        /// <param name="hostingContext">The hosting context.</param>
+        public static IConfigurationBuilder AddAzureAppConfiguration(
```

Comment:
Add info about configuration schema into `<remarks></remarks>` and add basic info about using this extension into `readme.md` file.

Reply:
added doc